addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12833,24 +12843,24 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#24116"
-msgid "Default TV Service"
+msgid "Default TV service"
```

Comment:
I never liked this - IMO this should be `Default TV show service` as it's in no way related to live TV or alike but only TV shows.